app/glossary.md

```diff
@@ -180,8 +201,11 @@ Teaching School Alliance
 UCAS
 : The Universities and Colleges Admissions Service whose main role is to operate the application process for British universities.
 
+UK ENIC
+: A service which provides impartial judgement on the international qualifications, and how they compare to UK qualifications. Previously known as UK NARIC prior to the UK leaving the European Union. ENIC stands for European Network of Information Centres, but the term does not need to be spelled out. See [UK ENIC](https://www.enic.org.uk) website.
+
 UK NARIC
-: All EU and EEA states (as well as some other associated countries) have a designated National Academic Recognition Information Centre that is responsible for recognising international qualifications and skills. The [National Recognition Information Centre for the United Kingdom](https://www.naric.org.uk/) is the UK’s agency. Providers can ask international applicants to provide a [statement of comparability](https://www.naric.org.uk/Qualifications/SOC/Default.aspx) to ensure their qualifications align with UK requirements.
+: Previous name for UK ENIC, whilst the UK was part of the EU.
```

Comment:
does this line need to be deleted?

Reply:
I figured it might might worth keeping it around for a bit longer, as we might still use it in a few places (in case anyone was familiar with the old name)?

Reply:
ah I see, yea fine!